draft-ietf-rtcweb-jsep.xml

```diff
@@ -1770,23 +1879,26 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
     	      <t>For RtpTransceivers that are still present, any
     	      "a=simulcast" line MUST stay the same.</t>
 
-              <t>If any MediaStreamTracks have been removed, either through
-              the removeStream method or by removing them from an added
-              MediaStream, their m= sections MUST be marked as recvonly by
-              changing the value of the <xref target="RFC3264"></xref>
-              directional attribute to "a=recvonly". The "a=msid", "a=ssrc",
-              and "a=ssrc-group" lines MUST be removed from the associated m=
-              sections.</t>
-
-              <t>If any MediaStreamTracks have been added, and there exist
-              m= sections of the appropriate media type with no associated
-              MediaStreamTracks (i.e. as described in the preceding paragraph),
-              those m= sections MUST be recycled by adding the new
-              MediaStreamTrack to the m= section. This is done by adding the
-              necessary "a=msid", "a=ssrc", and "a=ssrc-group" lines to the
-              recycled m= section, and removing the "a=recvonly" attribute.
-              </t>
-            </list></t>
+              <t>The directional attribute MAY change, depending on
```

Comment:
This is kind of confusing, as it seems to say that the value MAY change but MUST have a specific value.